### PR TITLE
Project restructuring

### PR DESCRIPTION
**Pull request by evleria**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,9 +5,10 @@ COPY go.mod .
 COPY go.sum .
 RUN go mod download
 
-COPY /cmd /cmd
-COPY /pkg /pkg
-RUN go build -o server ./cmd/rest/main.go
+COPY main.go /
+COPY docs /docs
```
Comment (unspectd) [inline Dockerfile]: COPY /docs /docs

File: Makefile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 swag:
-	swag init --parseDependency --parseDepth=5 -d ./cmd/rest -o ./cmd/rest/docs
+	swag init --parseDependency --parseDepth=5
```
Comment (unspectd) [inline Makefile:2]: Does it work without `--parseInternals`?

**Review by unspectd — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (evleria) [reply Dockerfile]: +

File: Makefile (hunk 2 above)
Comment (evleria) [reply Makefile:2]: Yes

**Pull request merged**